lib/article_json/import/google_doc/html/text_box_parser.rb

```diff
@@ -22,10 +22,21 @@ def content
             @nodes.map { |node| parse_sub_node(node) }.compact
           end
 
+          # Extract any potential tags, specified in brackets after the URL
```

Comment:
what URL? :thinking: 
```suggestion
          # Extract any potential tags, specified in brackets after the Textbox definition
```

Reply:
You are right, changed!

Reply:
From the test it looks like they come at the beginning of the box content?